GitHubCore.ps1

```diff
@@ -165,7 +165,7 @@ function Invoke-GHRestMethod
         $headers['Authorization'] = "token $AccessToken"
     }
 
-    if ($Method -in ('post', 'patch', 'put'))
+    if ($Method -in ('post', 'patch', 'put', 'delete'))
```

Comment:
Is there a way to persist this script variable inside the scriptBlock which runs in the background? I would think we would rather not pass this in as a variable otherwise.

Reply:
Yeah, that's a good catch.  I think for now, just ignore this suggestion.

Reply:
Disagree.  If the scriptblock is dependent on this variable then we simply pass it as input to the scriptblock, like any other dependency.

Reply:
Done!